src/main/java/duke/util/SaveTasks.java

```diff
@@ -0,0 +1,99 @@
+package duke.util;
+
+import duke.DukeException;
+import duke.task.*;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class SaveTasks {
+    private static final int TASKTYPE = 0;
+    private static final int ISTASKDONE = 1;
+    private static final int TASKDESCRIPTION = 2;
+    private static final int TASKDATETIME = 3;
+    private ArrayList<Task> savedTasks = new ArrayList<>(100);
+
+    private String fileDir;
+    private String filePath;
+
+    /**
+     * A constructor for SaveTasks
+     * @param fileDir the directory of the txt database file
+     * @param filePath the path to the txt database file
+     */
```

Comment:
From coding standard, Javadoc comment require empty line between description and parameter section, I have noticed this issue in other places as well.